pkg/common/config/log_metrics.go

```diff
@@ -23,14 +23,22 @@ var beforeSuiteMetrics = LogMetrics{}
 // GetLogMetrics will return the log metrics.
 func GetLogMetrics() LogMetrics {
 	once.Do(func() {
-		viper.UnmarshalKey("logMetrics", &logMetrics)
+		if CloudProvider.CloudProviderID == "gcp" {
```

Comment:
What if we removed the if statement and instead dynamically attempted to do UnmarshalKey? Like...

`err := viper.UnmarshalKey(fmt.Sprintf("%slogMetrics", CloudProvider.CloudProviderID)` and then if there's an error, log it and bubble it up?

Reply:
Yup, this makes more sense. I've made the changes as suggested. Thanks!